src/api/index.tsx

```diff
@@ -302,6 +306,17 @@ export const fetchAllPages = async <ResultsType,>(
   );
 };
 
+const makeApiQuery = (path?: string, fileOnly?: boolean) => {
+  const query: CallApiQuery = {};
+  if (path) {
```

Comment:
when would `path` ever be missing/empty?

Reply:
When you go from Reviewer Tools to Code Manager it takes you to the default file, which is generally `manifest.json`, but at that point no file is actually selected so no `path` is sent to the API.